src/types/Map.ts

```diff
@@ -102,8 +102,8 @@ export type MapEntityId = {
  * map entity options
  */
 export type MapEntityOptions = {
-  highlight?: MapEntityId;
-  focus?: MapEntityId;
+  highlight?: MapEntityId[];
+  focus?: MapEntityId[];
```

Comment:
Oops, I started out working on the zoom and highlight feature, but slid into just making the map interactive. I'll still use this. The intent was to allow the map to highlight/focus on more than one entity, which will be useful if, for example, search matches more than one zone.